### PR TITLE
Ref/intro apps

### PR DESCRIPTION
**Pull request by lheagy**
# First impressions
- add the case history image to the index page
image

# Apps
- more description on how to get up and running with Binder, Azure
image

# General cleanup
- update copyright year to 2018
- bibtex cleanup
- add live links that are slow to resolve to the linkcheck ignore

### CODE REVIEW

**Comment by lheagy** [comment]
before merging 
- [ ] coda patch
- [ ] release notes

File: apps.rst [hunk 1]
```diff
@@ -24,40 +21,122 @@ If you run into any issues, please let us know at: http://github.com/geoscixyz/e
...
+
+3. :ref:`Run the jupyter notebook <jupyter_notebooks>`
+
+.. image:: images/binder-steps.png
+    :alt: binder-steps
+    :width: 100%
+    :align: center
 
-.. image:: images/notebook_list.png
+
```
[the remark this answers is not in the document]
Comment (sgkang) [reply apps.rst]: Also, we need to set our python kernel consistently. 
For different python, we need to re-install em_examples on azure.

[the remark this answers is not in the document]
Comment (lheagy) [reply apps.rst]: @sgkang: as long as users check the box that says `clone recursively` then all of the dependencies in `requirements.txt` are installed (including `em_examples`). so the pip install is not necessary at the top of the notebooks

**Comment by lheagy** [comment]
Thanks @sgkang!

**Pull request merged**